syntax = "proto3";

message import2responsemsg {
}
message import2requestmsg {
}
